app/models/levelling_up_premium_payments/eligibility.rb

```diff
@@ -36,6 +36,20 @@ class Eligibility < ApplicationRecord
       "itt_academic_year" => ["eligible_itt_subject"]
     }.freeze
 
+    # Generates an object similar to
+    # {
+    #   <AcademicYear:0x00007f7d874293c8 @start_year=2017, @end_year=2018> => "2017/2018",
+    #   <AcademicYear:0x00007f7d87429288 @start_year=2018, @end_year=2019> => "2018/2019",
+    #   <AcademicYear:0x00007f7d87429260 @start_year=2019, @end_year=2020> => "2019/2020",
+    #   <AcademicYear:0x00007f7d87429238 @start_year=2020, @end_year=2021> => "2020/2021",
+    #   <AcademicYear:0x00007f7d87429210 @start_year=2021, @end_year=2022> => "2021/2022",
+    #   <AcademicYear:0x00007f7d87428c98 @start_year=nil, @end_year=nil> => "None"
+    # }
+    SELECTABLE_ITT_ACADEMIC_YEARS =
+      JourneySubjectEligibilityChecker.selectable_itt_years_for_claim_year(AcademicYear.for(Date.today)).each_with_object({}) do |year, hsh|
+        hsh[year] = AcademicYear::Type.new.serialize(year)
+      end.merge({AcademicYear.new => AcademicYear::Type.new.serialize(AcademicYear.new)})
```

Comment:
It's not obvious what this blank instance is used for. I would expect it to be the current year but that is returned by `selectable_itt_years_for_claim_year` so may now be redundant, as 2022 wasn't in the original implementation's list.

Reply:
Forgot to answer that the blank instance is for the `None of the above` option.